reflex/components/component.py

```diff
@@ -1301,24 +1301,29 @@ def _get_imports(self) -> imports.ImportDict:
         ]
 
         # If any subclass implements add_imports, merge the imports.
-        def _make_list(
-            value: str | ImportVar | list[str | ImportVar],
-        ) -> list[str | ImportVar]:
-            if isinstance(value, (str, ImportVar)):
-                return [value]
-            return value
-
-        _added_import_dicts = []
+        # def _make_list(
+        #     value: str | ImportVar | list[str | ImportVar],
+        # ) -> list[str | ImportVar]:
+        #     if isinstance(value, (str, ImportVar)):
+        #         return [value]
+        #     return value
+
+        _added_import_dicts: list[ParsedImportDict] = []
         for clz in self._iter_parent_classes_with_method("add_imports"):
-            _added_import_dicts.append(
-                {
-                    package: [
-                        ImportVar(tag=tag) if not isinstance(tag, ImportVar) else tag
-                        for tag in _make_list(maybe_tags)
-                    ]
-                    for package, maybe_tags in clz.add_imports(self).items()
-                }
-            )
+            added_imports_ = clz.add_imports(self)
+            if not isinstance(added_imports_, list):
+                added_imports_ = [added_imports_]
+            for added_imports in added_imports_:
+                _added_import_dicts.append(parse_imports(added_imports))
```

Comment:
the naming here is a bit hard to read because `added_imports_` and `added_imports` are very similar looking

maybe you can use `list_of_import_dicts` instead of `added_imports_` and `import_dict` instead of `added_imports`?